src/docs/goals/platforms.md

```diff
@@ -3,18 +3,44 @@ title: "Platform"
 featured: ../images/featured/goals.png
 ---
 
-## WORK IN PROGRESS NOTES
-
-- Synthesize the product function
-- Justifying funding for the platform team can be difficult as it is indirect business value
-- AuxEng allows you to staff a platform team up with senior talent while still directly contributing to business outcomes
-- Knowing the right problems to solve can be difficult. Being embedded brings challenges, low hanging fruit to the surface.
-  - Impossible for a senior engineer to know what it's like to be a junior engineer, or not know the platform
-  - Seeing a junior engineer struggle makes points of friction clear
-  - Engineers experiencing pain don't always realize when something is easy to improve.
-- Giving AuxEngineers 1 day per week to make self-directed investments / explorations in the platform gives them is powerful
-  - Self-direction is powerful. Process adds friction
-  - Short feedback loops. Experience pain -> Solve problem
-  - Writing documentation can be draining, but not nearly as much when you are solving a problem just experienced first hand
-    - You feel like you're helping someone
-    - That person validates and expresses gratitude. Feels rewarding
+In Auxiliary Engineering, we move out to consumer teams from a product team to solve problems and learn more about our products. The tools and products that an engineering support team puts out are that team's _platform_. Our goal in doing this is to enrich the platform, and the experience of those using it.
+
+A Python Platform team would provide tools that make Python simple to use and operate. A team focused on testing may maintain a continuous integration pipeline infrastructure, or testing frameworks. A Kubernetes-centric team may provide templates and automation to make interacting with Kubernetes simple for critical team operations and application development.
+
+Our goals through AuxEng provide benefits to any team developing or maintaining a platform.
+
+## Product, Platform, AuxEng
```

Comment:
We should use consistent spacing in reference to Aux Eng / Aux Engineers throughout the article:

```suggestion
Our goals through Aux Eng provide benefits to any team developing or maintaining a platform.

## Product, Platform, Aux Eng
```